src/node_options.cc

```diff
@@ -661,6 +661,9 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "run test at specific shard",
             &EnvironmentOptions::test_shard,
             kAllowedInEnvvar);
+  AddOption("--test-skip-pattern",
```

Comment:
please add to `node.1` as well https://github.com/nodejs/node/blob/main/doc/node.1